polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -264,7 +264,7 @@ object PythonInterpreter {
 class PythonDisplayHook(out: Enqueue[IO, Result]) {
   private var current = ""
   def output(str: String): Unit =
-    if (str != null) out.enqueue1(Output("text/plain", str)).unsafeRunSync()
+    if (str != null) out.enqueue1(Output("text/plain; lang=python", str)).unsafeRunSync()
```

Comment:
Hmm, so what does `lang` mean? Does it mean that the output originated from that language, or that the output is text *in that language*? Because here, all python output is going to be marked `lang=python`, even from `print` etc.